simplq/src/store/asyncActions/deleteQueue.js

```diff
@@ -0,0 +1,31 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import useAuth, { makeAuthedRequest } from 'api/auth';
+import * as RequestFactory from 'api/requestFactory';
+
+const typePrefix = 'deleteQueue/requestStatus';
+
+/**
+ * A hook to access the deleteQueue async action creator.
+ *
+ * @returns — deleteQueue async async action creator
+ */
+const useDeleteQueue = () => {
+  const auth = useAuth();
+
+  const deleteQueue = createAsyncThunk(typePrefix, async (arg) => {
+    if (!auth || !auth.isAuthenticated) {
+      return { queues: [] };
+    }
```

Comment:
Here we should check just for auth.isLoading, this should work as in case the user has not logged in, it will send "Bearer anonymous" for now. Later I will handle it along with https://github.com/SimplQ/simplQ-frontend/issues/477